tests_integration/test_data/projects/streamlit_v2/snowflake.yml

```diff
@@ -2,7 +2,8 @@ definition_version: 2
 entities:
   my_streamlit:
     type: "streamlit"
-    name: test_streamlit_deploy_snowcli
+    identifier:
+      name: test_streamlit_deploy_snowcli
```

Comment:
Everywhere you changed name -> identifier, but here not. Why?

Reply:
To make testing easier, the name is altered during tests. And it's easier to alter it as part of identifier than as single string.